.ruby-version

```diff
@@ -1 +1 @@
-3.3.4
+3.3.4
```

Comment:
seems like we messed with the newline in this file, possibly unintentionally?

Reply:
def uninintentional